### PR TITLE
feat(shared-data): Add configurations for EVT pipettes

### PR DESCRIPTION
**Pull request by andySigler**
# Overview

Makes testing easier for EVT pipettes by adding configs for new model numbers.

# Changelog

- Adds models:
  - `p1000_single_v3.3`
  - `p50_single_v3.3`
  - `p50_single_v4.3`
- Updates plunger positions for all EVT units
- Lowers plunger currents for all GEN3 pipettes (to avoid ever overheating again)
- Add extra `ul_per_mm` tables for the `p1000_single_gen3` to enable testing of these functions

# Review requests

# Risk assessment

### CODE REVIEW

**Comment by andySigler** [comment]
@Laura-Danielle  I've moved the extra functions for the P1000 to the `hardware-testing` package, so we can continue testing without these additional functions affecting production code 👍

File: api/tests/opentrons/config/test_pipette_config.py [hunk 1]
```diff
@@ -40,6 +40,7 @@ def check_sequences_close(
             or c.endswith("1.6")
             or "v2" in c
             or "v3" in c
+            or "v4" in c
```
Comment (sfoster1) [inline api/tests/opentrons/config/test_pipette_config.py:43]: what's this for?

**Review by sfoster1 — APPROVED** [review]
Looks good to me but I think you can remove that change to `test_pipette_config`.

File: api/tests/opentrons/config/test_pipette_config.py (hunk 1 above)
Comment (andySigler) [reply api/tests/opentrons/config/test_pipette_config.py:43]: it was failing with out this. This test seems to be making sure that aspirate function versions are properly set on pipettes with multiple versioned functions. That `if not` block was accidentally accumulating GEN3 pipettes, which  were then breaking the test 🤷

**Pull request merged**